mopidy_gmusic/commands.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import print_function
+from oauth2client.client import OAuth2WebServerFlow
+import gmusicapi
+from mopidy import commands
+
+
+class GMusicCommand(commands.Command):
+    def __init__(self):
+        super(GMusicCommand, self).__init__()
+        self.add_child('login', LoginCommand())
+
+
+class LoginCommand(commands.Command):
+    def run(self, args, config):
+        oauth_info = gmusicapi.Mobileclient._session_class.oauth
+        flow = OAuth2WebServerFlow(**oauth_info._asdict())
+        print('')
```

Comment:
```suggestion
        print()
```

This provides the exact same behavior.